lib/builder/webpack/client.js

```diff
@@ -147,6 +148,15 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
       )
     }
 
+    if (!this.options.dev && this.options.build.extractCSS) {
+      config.plugins.push(
```

Comment:
This can use another plugin which is suggested by sorka and be put into optimization.minifier https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/README.md#minimizing-for-production

Reply:
Will look into it this afternoon ☺️

Reply:
Agree with @clarkdo too. [NMFR/optimize-css-assets-webpack-plugin](https://github.com/NMFR/optimize-css-assets-webpack-plugin) would be a better idea.

Also, we are already using cssnano (via postcss) for production build:

https://github.com/nuxt/nuxt.js/blob/e9bb9e7236eb0c9643daee8e557e6e254a08db8e/lib/builder/webpack/utils/postcss.js#L47

Reply:
Thanks for the feedback 👌 I'll swap it then.

Yes, we use `cssnano`, but as it seems it will be applied **before** extraction and won't minify extracted CSS therefore. You can run the test without my changes and see it failing (or check [this file](https://developmint.de/_nuxt/2183bc8206ca1adc17d7.css) which was extracted and not minified ([this repo](https://github.com/Developmint/developmint.de/))

Reply:
@manniL Should be `this.options.build.extractCSS` ?

Reply:
Dang, right! :see_no_evil: 
It goes well now, thx.

People could still override the whole minimizer, couldn't they?
If not, I'd agree with the extract flag there :relaxed:

Reply:
Sure! There you go :relaxed: